ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerViewManager.m

```diff
@@ -159,8 +159,8 @@ - (void)bannerViewDidDismissScreen:(GADBannerView *)bannerView {
   [self sendEvent:@"onAdClosed" payload:nil];
 }
 
-- (void)bannerView:(GAMBannerView *)bannerView
```

Comment:
Actually we also need the bannerView method in case someone uses a GAMBannerView (Google Ad Manager Banner) instead of the regular GADBannerView. So please leave this method and add a new adView method below.

Reply:
See: https://developers.google.com/ad-manager/mobile-ads-sdk/ios/banner#app_events